packages/ui/src/components/flag/flag.native.tsx

```diff
@@ -41,7 +40,7 @@ export function Flag({
       alignItems={getFlagAlignment(align)}
       style={[boxProps]}
     >
-      <Box flex={1} marginLeft={reverse ? spacing : '0'} marginRight={!reverse ? spacing : '0'}>
+      <Box marginLeft={reverse ? spacing : '0'} marginRight={!reverse ? spacing : '0'}>
```

Comment:
I think this `flex={1}` is necessary for it to render properly. You need to give things at least a `flex={1`. 

You can see in storybook without it:
![Screenshot 2024-09-06 at 05 51 34](https://github.com/user-attachments/assets/7ecb73b7-0142-45e8-b4e1-38a6e0740b42)

This is the storybook with `flex={1`
![Uploading Screenshot 2024-09-06 at 05.51.06.png…]()

Reply:
This is what I encountered trying to use `ItemLayout` with `Flag` in code, did you actually use it anywhere?

![signal-2024-09-06-084504_002](https://github.com/user-attachments/assets/8576bbd9-0939-4cc4-9722-34e0aaf53234)

Our web version of Flag doesn't behave that way? If I don't remove it, what else do you suggest for the Avatar here?

Reply:
Flag... https://csswizardry.com/2013/05/the-flag-object/

Reply:
I was building `Flag` based on web and know that link. 

I had it working but the only place I was using it was in `ItemLayout` so far. This is the original PR I did:
https://github.com/leather-io/mono/pull/383 

I just know that not having this `flex={1}` has broken the display in the story. 

Are you using it anywhere? I was planning to use it in [the headers ](https://github.com/leather-io/issues/issues/230) so I can fix this then and let you merge this

Reply:
I am using it, yes, on the settings `Display` screen. It had to be changed to work. The web version, if you are following it, doesn't have the flex 1 on it. That is why I posted the link and mentioned it. I don't think you are building a Flag correctly?

My change (and web version w/out flex 1):
![IMG_1689](https://github.com/user-attachments/assets/317379b1-28f4-4271-87bf-36b0832662d7)

Your version with flex 1:
![IMG_1690](https://github.com/user-attachments/assets/a33b8ed1-e3c0-4129-bb71-f91f467c79b2)

If I put a fixed width/height on the Avatar:
![IMG_1691](https://github.com/user-attachments/assets/0fd47378-c162-4444-97aa-f1efbd02a86a)

What am I missing? Maybe the story you composed is the 'broken' part? Why are you feeling like the image part of the Flag should flex?

Reply:
@kyranjamie you built the web Flag, correct? Thoughts here?

Reply:
The idea of Flag is to simply position text next to an image. The implementation can vary, flex, float, grid etc, so long as the it doesn't make assumptions of the content inside.

As far as I understand flex: 1, this sizes an element relative to its siblings, which is not an assumption Flag should make.

Reply:
Yeah, I did this in native story that is similar to implementation in web story...

<img width="1159" alt="Screenshot 2024-09-06 at 12 54 16 PM" src="https://github.com/user-attachments/assets/e6e74dc2-384a-404a-afb9-8cde6436bba5">

Reply:
That is with removing flex 1, so not really broken. @pete-watters let me know if you understand a prob with this?

Reply:
My understanding is that in React Native, you have to declare `flex={1` [for each view for it to appear](https://reactnative.dev/docs/flexbox), maybe that's incorrect though and children can force the parent to view if they have dimensions supplied.  

I can see it is working in your updated story, passing a box with dimensions 50x50. Before, it wasn't showing the `Img` section - (the green part of the story) without using Flex = 1. It could have been that my demo `Box` for `img` was lacking this dimension. 

The Flag I implemented is basically the same as this but with a `flex=1` on the first element so I do know how to build it. It just looked broken when I checked storybook so flagged it. It looks fine now 👍